buildSrc/src/main/kotlin/com/strumenta/kotlinmultiplatform/gradle/ext/Project.ext.kt

```diff
@@ -70,61 +79,3 @@ fun Project.mavenRepositoryUrl(): String? {
  */
 fun Project.targetsNative(): Boolean =
   booleanProperty("target.is.native")
-
-fun Project.targetsJS(): Boolean =
-  booleanProperty("target.is.js")
-
-
-fun PublishingExtension.addSonatypeRepository(project: Project) {
```

Comment:
If we remove this configuration, where do we setup the credentials for publishing to sonatype?

Reply:
Oh I removed that one because it was unused, couldn't find any reference to it.